### PR TITLE
"classic" -> "curly" in syntax-conversion-guide

### LINKED ISSUE #929: Change "classic invocation" to "curly invocation" in angle bracket syntax conversion guide
Currently the [angle bracket syntax conversion guide](https://guides.emberjs.com/release/reference/syntax-conversion-guide/#toc_angle-bracket-syntax) uses the term "classic invocation" to refer to curly invocation, unfortunately this conflicts pretty strongly with what we decided was "classic components".

Specifically, "classic components" means components extending from `Ember.Component`, it is pretty confusing to have "classic invocation" to use a different definition of "classic". For example, you can absolutely invoke `@glimmer/component` based components with curlies but they are _for sure_ not "classic components".

We should change usages of "classic" in the page I linked above to "curly" or "curlies" (based on usage).

### PR DESCRIPTION
**Pull request by bantic**
(This PR targets the `octane` branch).

And some light copy-editing.

Unified capitalization of "angle bracket syntax" to be lowercased throughout.

Fixes #929

### CODE REVIEW

**Comment by bantic** [comment]
Great! I'm happy to switch the target for the PR if needed. If so, or if there's anything else I should do here, please let me know! thanks.

**Comment by jenweber** [comment]
Hi, thanks for this PR!

We discussed this at the learning team meeting and we concluded that we'll move forward with referring to these as "curly braces." The rationale is that we did show angle brackets vs curly braces language long before we came up with "classic" terminology. We do want to say "curly braces" and not just "curly." This is kinda inconsistent at the moment across the guides but it's where we want to go.

I'm adding this as code suggestions and then it will be good to go.

File: guides/release/reference/syntax-conversion-guide.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
 ## Angle Bracket Syntax
 
-There are two ways to invoke a component in a template: classic invocation syntax (`{{my-component}}`), and [angle bracket invocation syntax](https://github.com/emberjs/rfcs/blob/master/text/0311-angle-bracket-invocation.md) (`<MyComponent />`).
+There are two ways to invoke a component in a template: curly invocation syntax (`{{my-component}}`), and [angle bracket invocation syntax](https://github.com/emberjs/rfcs/blob/master/text/0311-angle-bracket-invocation.md) (`<MyComponent />`).
```
Comment (jenweber) [inline guides/release/reference/syntax-conversion-guide.md]:
```suggestion
There are two ways to invoke a component in a template: curly braces invocation syntax (`{{my-component}}`), and [angle bracket invocation syntax](https://github.com/emberjs/rfcs/blob/master/text/0311-angle-bracket-invocation.md) (`<MyComponent />`).
```

File: guides/release/reference/syntax-conversion-guide.md [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
 The difference between them is syntactical.
-Classic invocation syntax may also be referred to as curly invocation syntax.
 
-**Classic invocation syntax:**
+**Curly invocation syntax:**
```
Comment (jenweber) [inline guides/release/reference/syntax-conversion-guide.md]:
```suggestion
**Curly braces invocation syntax:**
```

File: guides/release/reference/syntax-conversion-guide.md [hunk 3]
```diff
@@ -17,24 +16,24 @@ Classic invocation syntax may also be referred to as curly invocation syntax.
...
 
-While the Angle Bracket Syntax may remind you of HTML elements, it comes with differentiating features such as using the `@` syntax for passing in arguments which sets it apart from traditional HTML elements easily.
+While the angle bracket syntax may remind you of HTML elements, it comes with differentiating features such as the `@` syntax for passing in arguments that sets it apart from traditional HTML elements.
 
 ### When to use angle bracket invocation syntax?
 
 The angle bracket invocation syntax is useful when you wish to pass arbitrary HTML attributes to the component.
 This is possible because in angle bracket invocation syntax there is a distinction between passing a named argument and an HTML attribute,
-while in classic invocation syntax everything is an argument to the component, either named or positional.
+while in curly invocation syntax everything is an argument to the component, either named or positional.
```
Comment (jenweber) [inline guides/release/reference/syntax-conversion-guide.md]:
```suggestion
while in curly braces invocation syntax everything is an argument to the component, either named or positional.
```

File: guides/release/reference/syntax-conversion-guide.md [hunk 4]
```diff
@@ -54,14 +53,14 @@ Block form components also follow the same pattern as HTML elements where an HTM
 
 ### Positional parameters
 
-The classic invocation syntax supports passing arguments to the component by their position.
-In the following example, `"greeting"` and `"name"` are positional parameters:
+The curly invocation syntax supports passing arguments to the component by their position.
```
Comment (jenweber) [inline guides/release/reference/syntax-conversion-guide.md]:
```suggestion
The curly braces invocation syntax supports passing arguments to the component by their position.
```

File: guides/release/reference/syntax-conversion-guide.md [hunk 5]
```diff
@@ -98,15 +97,17 @@ export default Component.extend({
...
 <MyGreeting @params={{array "Hello" "World"}}>
 ```
 
-### When to use classic invocation syntax?
+### When to use curly invocation syntax?
 
-Although Angle Bracket syntax is considered to be the best approach, classic invocation syntax is fine to keep using.
-In some cases, classic invocation is still required. 
-When you need direct support for positional arguments or if your components are nested within the file tree, you should still reach for those curly braces:
+Although angle bracket syntax is considered to be the best approach, curly invocation syntax is fine to keep using.
```
Comment (jenweber) [inline guides/release/reference/syntax-conversion-guide.md]:
```suggestion
Although angle bracket syntax is considered to be the best approach, curly braces invocation syntax is fine to keep using.
```

File: guides/release/reference/syntax-conversion-guide.md [hunk 6]
```diff
@@ -98,15 +97,17 @@ export default Component.extend({
...
+In some cases, curly invocation is still required.
```
Comment (jenweber) [inline guides/release/reference/syntax-conversion-guide.md]:
```suggestion
In some cases, curly braces invocation is still required. 
```

**Comment by bantic** [comment]
@jenweber Thanks for the review. I accepted all the suggestions, let me know if anything else is needed. :)

**Pull request merged**